VoodooInput/VoodooInputSimulator/VoodooInputSimulatorDevice.cpp

```diff
@@ -345,6 +326,10 @@ void VoodooInputSimulatorDevice::releaseResources() {
     OSSafeReleaseNULL(work_loop);
     
     OSSafeReleaseNULL(new_get_report_buffer);
+    
+    kernel_buffer->complete();
```

Comment:
This needs a check whether `kernel_buffer` is NULL or not before accessing the entry.

Reply:
I've adderssed this issue in start() function.